src/common/librlist/rlist.c

```diff
@@ -939,7 +1102,8 @@ struct rlist *rlist_from_R (const char *s)
     struct rlist *rl = NULL;
     json_t *o = json_loads (s, 0, &err);
     if (o)
-        rl = rlist_from_json (o, &err);
+        if (!(rl = rlist_from_json (o, &err)))
+            fprintf (stderr, "%s", err.text);
```

Comment:
Just checking... do we really want this to go `stderr`?

Reply:
That's just some leftover debug. Will remove.